broker/src/test/java/io/zeebe/broker/test/EmbeddedBrokerConfigurator.java

```diff
@@ -20,7 +20,7 @@
 public final class EmbeddedBrokerConfigurator {
 
   public static final Consumer<BrokerCfg> DEBUG_EXPORTER =
-      cfg -> cfg.getExporters().put("DebugLogExporter", DebugLogExporter.defaultConfig(false));
+      cfg -> cfg.getExporters().put("DebugLogExporter", DebugLogExporter.defaultConfig());
```

Comment:
Is it deliberate that the literal is used here? (as opposed to using `DebugLogExporter.defaultConfig();`

Reply:
I don't understand - do you mean the exporter ID? This is using `DebugLogExporter.defaultConfig()`.

Reply:
I mean the key in the map

Reply:
Ah, no I just refactored the usage - I can't really speak for the original intent, and I was unaware we had a `DebugLogExporter.defaultExporterId()`, but I can use it :+1: